docs/source/usersguide/tallies.rst

```diff
@@ -0,0 +1,311 @@
+.. _usersguide_tallies:
+
+==================
+Specifying Tallies
+==================
+
+.. currentmodule:: openmc
+
+In order to obtain estimates of physical quantities in your simulation, you need
+to create one or more tallies using the :class:`openmc.Tally` class. As
+explained in detail in the :ref:`theory manual <methods_tallies>`, tallies
+provide estimates of a scoring function times the flux integrated over some
+region of phase space, as in:
+
+.. math::
+
+    X = \underbrace{\int d\mathbf{r} \int d\mathbf{\Omega} \int
+    dE}_{\text{filters}} \underbrace{f(\mathbf{r}, \mathbf{\Omega},
+    E)}_{\text{scores}} \psi (\mathbf{r}, \mathbf{\Omega}, E)
+
+Thus, to specify a tally, we need to specify what regions of phase space should
+be included when deciding whether to score an event as well as what the scoring
+function (:math:`f` in the above equation) should be used. The regions of phase
+space are called *filters* and the scoring functions are simply called *scores*.
+
+-------
+Filters
+-------
+
+To specify the regions of phase space, one must create a
+:class:`openmc.Filter`. Since :class:`openmc.Filter` is an abstract class, you
```

Comment:
The `Filter` class doesn't include a hyperlink, perhaps this is due to the fact that it is abstract?